src/extlibs/hrvo/simulator.cpp

```diff
@@ -45,11 +45,19 @@
 #include "software/logger/logger.h"
 
 HRVOSimulator::HRVOSimulator(float time_step, const RobotConstants_t &robot_constants)
-    : global_time(0.0f),
-      time_step(time_step),
+    : frame(0),
+      primitive_set(),
+      add_ball_agent(false),
+      ball_agent_id(-1),
       robot_constants(robot_constants),
+      global_time(0.0f),
+      time_step(time_step),
+      last_time_velocity_updated(0.0f),
       reached_goals(false),
-      kd_tree(std::make_unique<KdTree>(this))
+      kd_tree(std::make_unique<KdTree>(this)),
+      agents(),
+      friendly_robot_id_map(),
+      enemy_robot_id_map()
```

Comment:
If we want to initialise everything, we should probably add `world` and `primitive_set` aswell.

Reply:
it should be there...